dev-docs/bidders/docereeadmanager.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: bidder
+title: Doceree AdManager
+description: Doceree AdManager Prebid Bidder Adapter
+pbjs: true
+biddercode: docereeadmanager
+media_types: banner
+gvl_id: 1063
+tcf2_supported: true
+tcfeu_supported: true
+sidebarType: 1
```

Comment:
Please add more meta fields to ensure publishers can better vet your adapter

- schain_supported: true/false
- dchain_supported: true/false
- userId: (list of supported vendors)
- safeframes_ok: true/false
- deals_supported: true/false
- floors_supported: true/false
- fpd_supported: true/false
- pbs: true/false
- prebid_member: true/false
- ortb_blocking_supported: true/partial/false
- privacy_sandbox: no or comma separated list of `paapi`, `topics`

see https://docs.prebid.org/dev-docs/bidder-adaptor.html#submitting-your-adapter for a full list